package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "last-train-to-nodeville",
+  "version": "1.0.0",
+  "description": "In the initial commit to this project, you have been provided with files in an initial commit that contain a JavaScript module and some associated tests. Your task is to use NPM to setup the necessary project dependencies to make it possible for the command `npm test` to successfully run the test suite.",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha tests.js"
```

Comment:
Nicely done, most people miss the part about the pattern we discuss on slide 52. Gotta tell mocha where to find the tests when you call it.